Python/Product/EnvironmentsList/PipExtensionProvider.cs

```diff
@@ -502,15 +484,21 @@ public class PipException : Exception {
             : base(info, context) { }
     }
 
-    public class ValueEventArgs<T> : EventArgs {
-        public T Value { get; set; }
+    public sealed class QueryShouldElevateEventArgs : EventArgs {
+        public bool Cancel { get; set; }
+        public bool Elevate { get; set; }
+        public string TargetDirectory { get; private set; }
```

Comment:
Don't need private set if you only assign in the constructor.  Same for class below.